### PR TITLE
feat: log new-line so user can differentiate sections better in console

### PR DESCRIPTION
**Pull request by BorntraegerMarc**
### Background

I always had a hard time reading next steps from Auto-GPT. Especially when I let it run in continuous mode and then checked the result after some time: Reading up on what it has done was difficult:
e.g:
Screenshot 2023-04-15 at 15 21 59

### Changes

Now there are new lines written for every new action. This way it's easier to read for the human eye:

Screenshot 2023-04-15 at 15 19 24

### Documentation

Documented in code.

### Test Plan

See screenshots above. I expect manual testing this small change is sufficient.

### PR Quality Checklist
- [X] My pull request is atomic and focuses on a single change.
- [X] I have thoroughly tested my changes with multiple different prompts.
- [X] I have considered potential risks and mitigations for my changes.
- [X] I have documented my changes clearly and comprehensively.
- [X] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Review by nponeccop — APPROVED** [review]

File: autogpt/agent/agent.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def start_interaction_loop(self):
                 # Get key press: Prompt the user to press enter to continue or escape
                 # to exit
                 self.user_input = ""
+                logger.typewriter_log("\n") # First log new-line so user can differentiate sections better in console
```
Comment (richbeales) [inline autogpt/agent/agent.py]: Should we use os.linesep rather than \n - does this work on win + linux/mac?

Comment (BorntraegerMarc) [reply autogpt/agent/agent.py]: This code will also work for Windows.
`os.linesep` is used more when writing to binary files. Not for writing to stdout (as our logger does). The logger uses `print()`.

> Writing code that runs on both Windows and Unix systems always has challenges but thankfully worrying about printing \n versus \r\n isn’t one of them. The print() function is smart enough to convert \n to \r\n when running on Windows.

Source: https://www.idtech.com/blog/what-is-n-in-python

**Comment by nponeccop** [comment]
@BorntraegerMarc There are conflicts now

**Review by nponeccop — DISMISSED** [review]

**Review by richbeales — APPROVED** [review]

**Comment by BorntraegerMarc** [comment]
I don't get it: What's the problem with check `Pull Request auto-label / conflicts (pull_request_target)`? Merge conflicts were resolved...

**Comment by BorntraegerMarc** [comment]
Updated my branch with master and now it seems to work...
If someone could give approval for the workflows, then I can merge my PR

**Comment by BorntraegerMarc** [comment]
Can a maintainer approve the workflow, so we can get this merged?

**Comment by p-i-** [comment]
This is a mass message from the AutoGPT core team.
Our apologies for the ongoing delay in processing PRs.
This is because we are re-architecting the AutoGPT core!

For more details (and for infor on joining our Discord), please refer to:
  https://github.com/Significant-Gravitas/Auto-GPT/wiki/Architecting

**Review by ntindle — APPROVED** [review]

**Pull request merged**